ingestion_server/ingestion_server/indexer.py

```diff
@@ -525,6 +525,8 @@ def create_and_populate_filtered_index(
             # 10k derived from in-production testing
             # See https://github.com/WordPress/openverse/issues/2963
             requests_per_second=15_000,
+            # Temporary workaround to allow the action to complete.
+            request_timeout=48 * 3600,
```

Comment:
I find the factorisation a bit weird.
```suggestion
            request_timeout=2 * 24 * 60 * 60,
```

Reply:
That's funny because I copied your own style to keep it even! 😆 I think I like more this short form.

https://github.com/WordPress/openverse/blob/e061bbc18d80b5c81993f68cc633c1840f12c035/ingestion_server/ingestion_server/es_helpers.py#L72